webapp/components/experiments/JsonViewer.js

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from 'react';
+import ReactJson from 'react-json-view';
+import { getParameters } from 'root/api/experiments';
+import Breadcrumbs from '@material-ui/core/Breadcrumbs';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+import {
+  fontColor, experimentLabelColor, bgDarker, bgDarkest, primaryColor,
+} from '../../theme';
+
+const useStyles = (theme) => ({
+  root: {
+    width: 'auto !important',
+    flexDirection: 'column',
+    backgroundColor: bgDarker,
+    padding: theme.spacing(2),
+    borderRadius: '0.25rem',
+    "& .MuiBreadcrumbs-separator": {
+      margin: theme.spacing(0),
+    },
+    '& .react-json-view': {
+      padding: theme.spacing(2),
+      overflow: 'auto',
+      borderRadius: '0.25rem',
+      '& span': {
+        opacity: 1,
+      },
+    },
+    '& .MuiBreadcrumbs-ol': {
+      paddingBottom: theme.spacing(2),
+      '& .MuiBreadcrumbs-li': {
+        '& a': {
+          color: experimentLabelColor,
+          fontSize: '1rem',
+        },
+        '& p': {
+          color: fontColor,
+          fontSize: '1rem',
+        },
+      },
+    },
+  },
+});
+
+const JsonViewer = (props) => {
```

Comment:
The JsonViewer is not necessarily specific to the Experiments or a Trial, I would like to keep the Viewer generalized, we also plan to use the JSON Viewer for other purposes.

This component fits well into the `components/general` package.

Change suggestion:
* JSON Object directly as input, the useEffect to fetch the parameters is better handled outside of the Viewer, makes it easier to reuse JsonViewer for different things
* Remove trial or experiment names in JsonViewer
* Show line numbers in viewer (probably offered by ReactJson as boolean prop?)